molo/polls/tests/base.py

```diff
@@ -57,3 +59,6 @@ def setUp(self):
 
         self.mk_main2(title='main3', slug='main3', path=00010003)
```

Comment:
`path` should be a string

Reply:
Will make a change separately to get this merged in now.